lib/chef/knife/core/windows_bootstrap_context.rb

```diff
@@ -75,13 +65,18 @@ def config_content
           client_rb = <<~CONFIG
             chef_server_url  "#{chef_config[:chef_server_url]}"
             validation_client_name "#{chef_config[:validation_client_name]}"
-            file_cache_path   "#{ChefConfig::Config.var_chef_dir(true)}/cache"
-            file_backup_path  "#{ChefConfig::Config.var_chef_dir(true)}/backup"
```

Comment:
These were changed to accept user's input but Windows was not updated to match: https://github.com/chef/chef/pull/9361